webapp/mapTest4.html

```diff
@@ -342,16 +342,87 @@
               .addTo(map); 
               }
               
-              function pressBtn9(){
```

Comment:
Isn't this function still called above? Line 103:  <li><a id="button_press9" onclick="pressBtn9()">Walter Gleason Gym</a></li>